/**:
  ros__parameters:
    # -- Node Configurations --
    planning_algorithm: "astar"  # options: astar, rrtstar
    waypoints_velocity: 5.0
    update_rate: 10.0
    th_arrived_distance_m: 1.0
    th_stopped_time_sec: 1.0
    th_stopped_velocity_mps: 0.01
    th_course_out_distance_m: 1.0
    vehicle_shape_margin_m: 1.0
    replan_when_obstacle_found: true
    replan_when_course_out: true

    # -- Configurations common to the all planners --
    # base configs
    time_limit: 30000.0
    minimum_turning_radius: 9.0
    maximum_turning_radius: 9.0
    turning_radius_size: 1
    # search configs
    theta_size: 144
    angle_goal_range: 6.0
    curve_weight: 1.2
    reverse_weight: 2.0
    lateral_goal_range: 0.5
    longitudinal_goal_range: 2.0
    # costmap configs
    obstacle_threshold: 100

    # -- A* search Configurations --
    astar:
      only_behind_solutions: false
      use_back: true
      distance_heuristic_weight: 1.0
